invenio_records_lom/resources/serializers/__init__.py

```diff
@@ -10,7 +10,7 @@
 
 from flask_resources.serializers import MarshmallowJSONSerializer
 from invenio_rdm_records.resources.serializers import UIJSONSerializer
-from lxml.builder import ElementMaker
+from lxml.builder import ElementMaker  # pylint: disable=no-name-in-module
```

Comment:
Is this really necessary? I thought that such disable statements are only necessary if pylint runs outside of a virtualenv where the packages are not installed?

Reply:
For performance reasons, some of `lxml`'s code is written in C.
C-code added attributes don't get picked up by pylint (not even if pylint is inside a fitting environment).